src/content/docs/usage/workspaces.mdx

```diff
@@ -79,7 +79,7 @@ Refer to __[Set the Default IDE](#set-the-default-ide)__ for more information.
 :::
 
 ## Delete a Workspace
-You can delete an existing Workspace.
+You can delete an existing Workspace interactively.
```

Comment:
As of https://github.com/daytonaio/daytona/releases/tag/v0.18.0, users can delete multiple workspaces in the TUI as well so please update accordingly

(They select multiple workspaces with `x` and confirm with `Enter`) 

P.S. If no workspaces are selected and the user presses Enter, it will delete the currently focused workspace

Reply:
Clarified in [35e694d](https://github.com/daytonaio/docs/pull/61/commits/35e694d8820179d667fa696b2c789c1380f494b8)